rollout/trafficrouting_test.go

```diff
@@ -238,7 +257,8 @@ func TestNewTrafficRoutingReconciler(t *testing.T) {
 			log:     logutil.WithRollout(r),
 		}
 		rc.defaultTrafficSplitVersion = "v1alpha1"
```

Comment:
Minor: In general, we have to be very careful about modifying structs that used in other tests otherwise they may have unintended consequences. For example, if this test was moved above the test that verifies the Unsupported TrafficSplit API Version error, that test would fail. As a result, we should declare our own controller object just for this test to prevent the global controller variable doesn't affect other tests. The tests are passing now, but it may cause a problems if these tests are refactored in the future and the developer doesn't realize that change.